sdk/appcenter-identity/src/main/java/com/microsoft/appcenter/identity/storage/TokenStorageFactory.java

```diff
@@ -7,6 +7,8 @@
 
 import android.content.Context;
 
+import com.microsoft.appcenter.utils.storage.AuthTokenStorage;
```

Comment:
was that to fix a Javadoc warning?

Reply:
No, this class was moved into another package (to core instead of identity module)